src/smartcontracts/smartContract.local.net.spec.ts

```diff
@@ -325,4 +330,51 @@ describe("test on local testnet", function () {
         queryResponse = await provider.queryContract(query);
         assert.equal(decodeUnsignedNumber(queryResponse.getReturnDataParts()[0]), 0);
     });
+
+    it("counter: should deploy and call using the SmartContractFactory", async function () {
```

Comment:
:rocket:  :+1: (in relation to the other comment about a next PR for integration tests, only now I see this test)